app/lib/efile/md/md502_calculator.rb

```diff
@@ -155,19 +155,19 @@ def calculate_line_a_primary
       end
 
       def calculate_line_a_spouse
-        filing_status_mfj? ? "X" : nil
+        state_filing_status_mfj? ? "X" : nil
       end
 
       def calculate_exemption_amount
         # Exemption amount
-        income_ranges = if filing_status_single? || filing_status_mfs?
+        income_ranges = if state_filing_status_single? || state_filing_status_mfs?
                           [
                             [-Float::INFINITY..100_000, 3200],
                             [100_001..125_000, 1600],
                             [125_001..150_000, 800],
                             [150_001..Float::INFINITY, 0]
                           ]
-                        elsif filing_status_hoh? || filing_status_mfj? || filing_status_qw?
+                        elsif state_filing_status_hoh? || state_filing_status_mfj? || state_filing_status_qw?
```

Comment:
with this change, a filer could get into the `else` for being a `:dependent` state_filing_status... is that what we want?